tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -647,6 +647,7 @@ Section Uninstall
 
   ; Delete app data
   ${If} $DeleteAppDataCheckboxState == 1
+    SetShellVarContext current
```

Comment:
This could lead into problems with `DeleteRegKey` on Line 657, lets move this logic section down before line 666

Reply:
done